lib/smart_app_launch/app_redirect_test.rb

```diff
@@ -126,6 +126,8 @@ def authorization_url_builder(url, params)
         oauth2_params
       )
 
+      add_message('info', "Inferno presented this URI for OAuth authorization: #{authorization_url}.")
```

Comment:
To clarify, I chose this language because Inferno doesn't redirect the browser; it just give the user a link to click. So I thought that might be confusing, especially since there is an _actual_ browser redirect here that's initiated by the server under test.

If you'd still prefer the language you suggested, that's also fine with me, but wanted to get my concern across. @Jammjammjamm

Reply:
Yeah, I prefer this language.

Reply:
Alright then. It's updated.